### PR TITLE
Transparent.png

### LINKED ISSUE #35: Partially transparent png
* `react-ideal-image` version: 0.0.4
* `node` version: 9.11.1
* `npm` (or `yarn`) version: 1.6.0

Relevant code or config

```javascript
```

What you did: I have an image that has a transparent portion

What happened: When the real image loads you can still see the lqip image

screen shot 2018-06-09 at 7 43 08 pm

Reproduction repository:

Problem description: Can still see lqip image.

Suggested solution: Disable lqip image on load.

### PR DESCRIPTION
**Pull request by hipstersmoothie**
closes #35 

**What**: Hide background-image on image load

**Why**: Transparent.png will show background image in transparent parts once loaded.

**How**: Add state to media that changes onLoad of image.

**Checklist**:

~~* [ ] Documentation~~ Probably unneeded
~~* [ ] Tests~~ No testing set up
* [x] Ready to be merged 
* [x] Added myself to contributors table

### CODE REVIEW

**Comment by hipstersmoothie** [comment]
failing build related to #31 ?

File: src/components/Media/index.js [hunk 1]
```diff
@@ -85,10 +89,17 @@ export default class Media extends PureComponent {
     ])
   }
 
+  onLoad = () => {
```
Comment (stereobooster) [inline src/components/Media/index.js]: you don't need onLoad here. This is controlled component it gets all state outside. You can use `props.icon === loaded`

File: src/components/Media/index.js [hunk 2]
```diff
@@ -139,7 +151,7 @@ export default class Media extends PureComponent {
       <div
         {...compose(
           theme.placeholder,
-          background,
+          this.state.showPlaceholder ? background : {},
```
Comment (stereobooster) [inline src/components/Media/index.js]:
```js
if (props.icon === loaded) {
  background = {}
} else if (placeholder.lqip) {
  background = {
    backgroundImage: `url("${placeholder.lqip}")`,
   }
...
```

**Comment by hipstersmoothie** [comment]
@stereobooster Addressed comments! 👍

**Comment by stereobooster** [comment]
Thank you

**Pull request merged**